docs/store/artifacts.md

```diff
@@ -130,5 +131,21 @@ Example artifact URLs:
 
 <!-- ## Dataset artifacts moved to data coll and prep, model and plot artifacts to working-with-data-and-model-artifacts -->
 
+## Deleting artifacts
+
+Artifacts are comprised of two parts: an artifact object that points to the artifact data; and the artifact data (files). You can delete artifacts from a specific project 
+and choose what you want to delete. Deleting artifacts is supported for v3IO, Google, Azure, DBFS, Filestore, and S3. 
+
+The options for {py:meth}`~mlrun.projects.MlrunProject.delete_artifact`:
```

Comment:
Why do we need all of the description here if we simply put a link to it: https://docs.mlrun.org/en/latest/api/mlrun.common/mlrun.common.schemas/mlrun.common.schemas.artifact.html#mlrun.common.schemas.artifact.ArtifactsDeletionStrategies

Reply:
It does not hurt to have it here